### ISSUE TITLE
Not Able to change the size of AdvanceEntry

### ISSUE THREAD

**Issue by ronakshethia** [enisn/Xamarin.Forms.InputKit#109]
i am trying to increase the height of entry but there is not effect of change on IOS

![image](https://user-images.githubusercontent.com/35444818/55318737-1886f680-5491-11e9-8e17-405119814abc.png)

**Comment by enisn** [comment]
I'll find the solution of your problem and update it

**Comment by hjavaher** [comment]
I'm seeing the same issue just creeped up. Updated to the latest version and just today noticed that the height was shrunk down to minimum

**Comment by ronakshethia** [comment]
is it fixed up now ?

**Comment by hjavaher** [comment]
I don't believe so, I can't seem to find a solution. Have been banging my head trying to find one. I'm sure @enisn will come through soon, he's amazing!

**Comment by hjavaher** [comment]
Any updates on this? Also, It would be awesome to get a brief release note with each update :)

**Comment by enisn** [comment]
> Any updates on this? Also, It would be awesome to get a brief release note with each update :)

I've just started finding solution of it

**Comment by ronakshethia** [comment]
Hi Enis,
Any Update with above issue fix ?

**Comment by hjavaher** [comment]
@enisn 
Any updates to this buddy. I was working just fine for me until I updated to the version around this time. I just updated to the latest and still same. All of the Inputs that I have is super slim and I can't make them regular size

**Comment by jiribanas** [comment]
If you set the **IconImage** property to a transparent PNG, the text field height will grow to accommodate the image. You can change the size of the image to make the text field smaller or larger. I am using it as a workaround until the issue is fixed.
